local.tf

```diff
@@ -95,6 +95,7 @@ locals {
     additional_ebs_volumes            = []                          # A list of additional volumes to be attached to the instances on this Auto Scaling group. Each volume should be an object with the following: block_device_name (required), volume_size, volume_type, iops, encrypted, kms_key_id (only on launch-template), delete_on_termination. Optional values are grabbed from root volume or from defaults
     additional_instance_store_volumes = []                          # A list of additional instance store (local disk) volumes to be attached to the instances on this Auto Scaling group. Each volume should be an object with the following: block_device_name (required), virtual_name.
     warm_pool                         = null                        # If this block is configured, add a Warm Pool to the specified Auto Scaling group.
+    timeouts                          = {}                          # A map of timeouts for create/update/delete operations
```

Comment:
should this be something like 
`timeouts                          = var.timeouts`

Reply:
only asking because #1552 created a `timeouts` variable in the variables.rb

Reply:
yeah, I did closed this and open this other one https://github.com/terraform-aws-modules/terraform-aws-eks/pull/1566, I did not reuse that variable but probably can add it as last default value